Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:10.15.0-alpine
+FROM node:14.16.0-alpine
 WORKDIR /app-prototype
 COPY package.json yarn.lock ./
-RUN npx yarn@1.12.1 install
+RUN npx yarn@1.12.1 install --production
```

Comment:
Speed up docker builds and make the resulting image smaller by not installing dev dependencies.